htdocs/modules/system/templates/system_userinfo.tpl

```diff
@@ -167,8 +167,10 @@
         <img src="<{$result.image}>" alt="<{$module.name}>"/>
         <strong><a href="<{$result.link}>" title="<{$result.title}>"><{$result.title}></a></strong>
         <br>
-        <span class="x-small">(<{$result.time}>)</span>
```

Comment:
We could simplify this by using the Smarty default modifier instead:

`<span class="x-small">(<{$result.time|default:''}>)</span>`

This would only require changing the templates, and there would be no need for the *if*.